lib/hpcbase.pm

```diff
@@ -53,8 +53,8 @@ sub get_remote_logs {
 
 sub switch_user {
     my ($self, $username) = @_;
-    type_string("su - $username\n");
-    type_string(qq/PS1="# "\n/);
+    enter_cmd("su - $username");
+    enter_cmd(qq/PS1="# "/);
```

Comment:
`type_string(qq/PS1="# "\n/);` is IMHO wrong candidate for `enter_cmd()` due `\n` not being at the end of the command.

Reply:
done